bionic/flow.py

```diff
@@ -1231,9 +1231,21 @@ def get(self, name, collection=None, fmt=None, mode=object):
             return set(values)
         elif collection is pd.Series or collection == "series":
             if len(result_group.key_space) > 0:
+                providers_by_name = {
+                    case_name: self._state.get_provider(case_name)
+                    for case_name in list(result_group.key_space)
```

Comment:
Ah you're right, we don't. I just copied what we were sending to `multi_index_from_case_keys` since I haven't used `ImmutableSequence` before. But going to remove this.